lib/tasks/ezid.rake

```diff
@@ -1,57 +0,0 @@
-require './lib/tasks/ezid/ezid_logger'
-
-namespace :ezid do
```

Comment:
I thought I have this file removed. This task is obsolete.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

On Fri, Sep 2, 2016 at 10:32 AM, pgwillia <notifications@github.com> wrote:

> In lib/tasks/ezid.rake
> <https://github.com/ualbertalib/HydraNorth/pull/1248#discussion_r77375139>
> :
>
> > @@ -1,57 +0,0 @@
> > -require './lib/tasks/ezid/ezid_logger'
> > -
> > -namespace :ezid do
>
> I completely missed this the first time through?!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/1248/files/245f7836c39128533fa466dd9c9309f52d654908..ab2547fee9bc06cffa5cf8a83bb82c0b42ce4aa5#r77375139>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB8-flV0RhBlQbezvsXelTZ_YHxlpcjQks5qmE-fgaJpZM4JwEUT>
> .
>